cluster/manifests/zmon-worker/deployment.yaml

```diff
@@ -97,6 +97,37 @@ spec:
                     name: zmon-worker-optional-config
                     key: WORKER_PLUGIN_HTTP_OAUTH2_TOKENS
                     optional: true
+            - name: WORKER_PLUGIN_REDIS_PASSWORD
+              valueFrom:
+                  configMapKeyRef:
+                    name: zmon-worker-optional-config
+                    key: WORKER_PLUGIN_REDIS_PASSWORD
+                    optional: true
+            - name: WORKER_PLUGIN_CASSANDRA_USER
+              valueFrom:
+                  configMapKeyRef:
+                    name: zmon-worker-optional-config
+                    key: WORKER_PLUGIN_CASSANDRA_USER
+                    optional: true
+            - name: WORKER_PLUGIN_CASSANDRA_PASS
+              valueFrom:
+                  configMapKeyRef:
+                    name: zmon-worker-optional-config
+                    key: WORKER_PLUGIN_CASSANDRA_PASS
+                    optional: true
+            - name: WORKER_PLUGIN_ELASTICSEARCH_URL
+              valueFrom:
+                  configMapKeyRef:
+                    name: zmon-worker-optional-config
+                    key: WORKER_PLUGIN_ELASTICSEARCH_URL
+                    optional: true
+            - name: WORKER_PLUGIN_KAIROSDB_URL
+              valueFrom:
+                  configMapKeyRef:
```

Comment:
Does this include a password? If yes, it should be in a secret.

Reply:
no, not supported